neurodamus/node.py

```diff
@@ -37,6 +37,8 @@
 # Internal Plugins
 from . import ngv as _ngv  # NOQA
 
+from neuron import h
```

Comment:
Please do not import neuron directly which will start machinery too early. Use the `Neurodamus` (`Nd`) wrapper

Reply:
Thank you, why is it fine to do so here:
https://github.com/BlueBrain/neurodamus/blob/8e388ba90dd196c03a6af63734fedab2f142c359/init.py#L9